sources/europe/fr/FR-BAN.geojson

```diff
@@ -10,6 +10,7 @@
         "url": "http://{switch:a,b,c}.layers.openstreetmap.fr/bano/{zoom}/{x}/{y}.png",
         "max_zoom": 20,
         "min_zoom": 12,
+	"overlay": true,
```

Comment:
(PS: there's two or three more places in other files of this PR where indention uses tabs instead of spaces.)